.github/workflows/build.yml

```diff
@@ -86,29 +86,29 @@ jobs:
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install cibuildwheel==2.21.2
 
-    # Patch: Fix versioning
-    - name: Download Patch 1/2
-      uses: suisei-cn/actions-download-file@818d6b7dc8fe73f2f924b6241f2b1134ca1377d9  # 1.6.0
-      id: setupversion
-      with:
-        url: "https://github.com/openPMD/openPMD-api/pull/1680.patch"
-        target: src/.patch/
-
-    # Patch: PYBIND11_FINDPYTHON=ON
-    - name: Download Patch 2/2
-      uses: suisei-cn/actions-download-file@818d6b7dc8fe73f2f924b6241f2b1134ca1377d9  # 1.6.0
-      id: setupfindpython
-      with:
-        url: "https://github.com/openPMD/openPMD-api/pull/1684.patch"
-        target: src/.patch/
-
-    - name: Apply Patches
-      run: |
-        cd src
-        git apply --exclude=.github/workflows/windows.yml .patch/1680.patch
-        git apply --exclude=cmake/dependencies/pybind11.cmake .patch/1684.patch
-        git fetch
-        git apply --exclude=setup.py -3 .patch/1684.patch
+#    # Patch: Fix versioning
+#    - name: Download Patch 1/2
+#      uses: suisei-cn/actions-download-file@818d6b7dc8fe73f2f924b6241f2b1134ca1377d9  # 1.6.0
+#      id: setupversion
+#      with:
+#        url: "https://github.com/openPMD/openPMD-api/pull/1680.patch"
+#        target: src/.patch/
+
+#    # Patch: PYBIND11_FINDPYTHON=ON
+#    - name: Download Patch 2/2
+#      uses: suisei-cn/actions-download-file@818d6b7dc8fe73f2f924b6241f2b1134ca1377d9  # 1.6.0
+#      id: setupfindpython
+#      with:
+#        url: "https://github.com/openPMD/openPMD-api/pull/1684.patch"
+#        target: src/.patch/
+
+#    - name: Apply Patches
+#      run: |
+#        cd src
+#        git apply --exclude=.github/workflows/windows.yml .patch/1680.patch
+#        git apply --exclude=cmake/dependencies/pybind11.cmake .patch/1684.patch
+#        git fetch
+#        git apply --exclude=setup.py -3 .patch/1684.patch
```

Comment:
The patches can be fully removed as a followup, these commits are part of the main source tree by now.